README.md

```diff
@@ -1,2 +1,11 @@
-# beekeeper-terraform
-Terraform deployment scripts for Beekeeper
+# Overview
+Terraform deployment scripts for [Beekeeper](https://github.com/ExpediaGroup/beekeeper).
+
+# Modules
+* [Main](https://github.com/ExpediaGroup/beekeeper/tree/master/main) - main AWS components required to run Beekepeer e.g. ECS, SQS, IAM etc.
+* [Lambda](https://github.com/ExpediaGroup/beekeeper/tree/master/lambda) - slack notifying Lambda which monitors Beekeeper's dead letter queue. This module is optional and not required to run Beekeeper.
```

Comment:
slack notifying Lambda which monitors Beekeeper's dead letter queue. -> Lambda which monitors Beekeeper's dead letter queue and sends notifications to Slack.

Reply:
Sounds good